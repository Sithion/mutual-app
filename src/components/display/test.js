import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Display from './display';

test('render "Display" correctly', () => {
  const container = renderer.create(
    <Display value="teste" />).toJSON();
  expect(container).toMatchSnapshot();
});
