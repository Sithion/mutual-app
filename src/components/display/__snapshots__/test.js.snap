// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render "Display" correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 15,
      }
    }
  >
    teste
  </Text>
</View>
`;
